src/main/java/org/greasemonkeys457/robot2018/RobotMap.java

```diff
@@ -8,18 +8,18 @@
      */
 
     // Drivetrain
-    public static int lfMotor = 1;
+    public static int lfMotor = 3;
```

Comment:
We should come up with a way to keep maps for both the competition bot and practice bot without removing the other.

Reply:
Or not.